pkg/kubelet/kubelet_server_journal_others.go

```diff
@@ -0,0 +1,40 @@
+//go:build !linux && !windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+// getLoggingCmd on unsupported operating systems returns the echo command and a warning message (as strings)
+func getLoggingCmd(a *journalArgs, services []string, boot int) (string, []string) {
+	return "echo", []string{"Operating System Not Supported"}
```

Comment:
Any particular reason for doing it this way, rather than just returning an error informing the OS is not supported?

Reply:
Mainly because the happy path (Copy() -> CopyForBoot() -> CopyServiceLogs()) did not involve any errors. This way the happy path is not affected and the error message would still be reflected back to the user in the unhappy path.

Reply:
I revisited this and the function now returns an error.